dashboard/config/levels/custom/music/musiclab_intro_repeat.level

```diff
@@ -0,0 +1,60 @@
+<Music>
+  <config><![CDATA[{
+  "published": true,
+  "game_id": 70,
+  "created_at": "2024-01-09T00:48:10.000Z",
+  "level_num": "custom",
+  "user_id": 976,
+  "properties": {
+    "encrypted": "false",
+    "background": "music-black",
+    "level_data": {
+      "library": "will",
+      "startSources": {
+        "blocks": {
+          "languageVersion": 0,
+          "blocks": [
+            {
+              "type": "when_run_simple2",
+              "id": "when-run-block",
+              "x": 30,
+              "y": 30
+            }
+          ]
+        }
+      },
+      "toolbox": {
+        "blocks": {
+          "Play": [
+            "play_sound_at_current_location_simple2"
+          ],
+          "Control": [
+            "repeat_simple2"
+          ]
+        },
+        "type": "flyout"
+      },
+      "text": "#### Repeat\n\nSave effort and use code to repeat a sound.\n\nUse the **repeat** block to play the same thing a few times.",
+      "validations": [
+        {
+          "conditions": [
+            {
+              "name": "played_sounds_together",
```

Comment:
Guessing we'll add validation for repeated sounds in a later change? Looks like currently you can pass the level just by playing a sound once.

Reply:
Yes, I figured I'd add the remaining validations in a follow-up.